GitFyle.Core.Api/Brokers/Loggings/ILoggingBroker.cs

```diff
@@ -12,5 +12,7 @@ public interface ILoggingBroker
         void LogTrace(string message);
         void LogDebug(string message);
         void LogWarning(string message);
+        void LogError(Exception exception);
```

Comment:
Remove LogError. The scope of this PR is only to add LogCritical. When we merge your other PR this will be added